app/commands/submission/representation/init.rb

```diff
@@ -28,7 +28,7 @@ def call
 
       def exercise_filepaths
         exercise_repo.tooling_filepaths.select do |filepath|
-          filepath.starts_with?(".meta")
+          filepath.starts_with?(".meta/")
```

Comment:
This is the only real exception, where we currently don't output the test/editor files for the representer.